tests/integration/update_cluster/launch_templates/data/aws_launch_configuration_master-us-test-1a.masters.launchtemplates.example.com_user_data

```diff
@@ -190,7 +190,7 @@ kubeAPIServer:
   - http://127.0.0.1:4001
   etcdServersOverrides:
   - /events#http://127.0.0.1:4002
-  image: k8s.gcr.io/kube-apiserver:v1.20.0
+  image: k8s.gcr.io/kube-apiserver:v1.21.0
```

Comment:
I think the `tests/integration/update_cluster/launch_templates` test was removed as part of #10937. The whole dir should be removed as part of #11206, but this PR should not touch any of the files.

Reply:
Files reverted in this PR and `tests/integration/update_cluster/launch_templates` directory removed in PR #11206.